pkg/ethutil/ethutil.go

```diff
@@ -106,6 +107,9 @@ func getInputIndex(
 		if err != nil {
 			return 0, fmt.Errorf("failed to parse input added event: %v", err)
 		}
+
+		slog.Debug("Input added", "event", inputAdded)
+
```

Comment:
Looking at this again, I think we should stay away from the ethutil package if the purpose of the change is simply logging the inputAdded event. 

I believe a better option would be simply getting the input index on the cli and using [`ethutil.GetInputFromInputBox()`](https://github.com/cartesi/rollups-node/blob/8b86538ea54f0dfca0fe8b64c26b90c3250bb02a/pkg/ethutil/ethutil.go#L122) to retrieve the info you want to log.

Reply:
This PR ended up with two different purposes. On would be to add verbose log, to check the steps on adding an Input. the other would be have a better feedback ( more data ) about the input. 

As the former was not fully implemented, I will go with your suggestion and make this Event Logging at the client code